client/verta/verta/deployment/resources.py

```diff
@@ -6,86 +6,77 @@
 from ..external import six
```

Comment:
This file's diff is a real disaster; I'd recommend just viewing the new file and reading that.